subprojects/common/test-okhttp/src/main/kotlin/com/avito/test/http/MockDispatcher.kt

```diff
@@ -16,15 +25,15 @@ class MockDispatcher(
 
         capturers.find { it.requestMatcher.invoke(request) }?.run {
             capture(request)
-            println("request captured: $request")
+            logger.info("request captured: $request")
```

Comment:
debug?

Reply:
there is no debug in java.util.Logger that is used here

Reply:
what about `com.avito.logger.Logger`? Is it for such kotlin libraries?

Reply:
Mockwebserver uses java.util under the hood, and it seems overcomplicated to redirect it to our logger